src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -116,34 +156,24 @@ public NodeSettings(Network innerNetwork = null, ProtocolVersion protocolVersion
         /// <param name="network">Specification of the network the node runs on - regtest/testnet/mainnet.</param>
         /// <param name="protocolVersion">Supported protocol version for which to create the configuration.</param>
         /// <returns>Default node configuration.</returns>
-        public static NodeSettings Default(Network network = null, ProtocolVersion protocolVersion = SupportedProtocolVersion)
+        public static NodeSettings Default(Network network = null, ProtocolVersion protocolVersion = SupportedProtocolVersion, string[] args = null)
```

Comment:
I don't think this method needs to have args as an argument.
I also think its body should be
`return new NodeSettings(innerNetwork: network, args: new string[0]);`

Reply:
There were existing test cases that called the Default method and then called LoadArguments. That means they were calling LoadArguments twice as the Default method contained a call to LoadArguments as well. The obvious resolution to that was to pass the arguments to the Default method so that it would not be necessary to call LoadArguments twice.

Reply:
Ah yes you're right.
I guess a resolution for this would be not to use `Default` method but rather to use the `NodeSettings` ctor?
Would that work?

Reply:
Argument removed.